docker-compose-generator/docker-fragments/groestlcoin-clightning.yml

```diff
@@ -3,30 +3,25 @@ version: "3"
 services:
   clightning_groestlcoin:
     image: groestlcoin/lightning:v0.10.0
-    stop_signal: SIGKILL
     container_name: btcpayserver_clightning_groestlcoin
     restart: unless-stopped
     environment:
       LIGHTNINGD_NETWORK: ${NBITCOIN_NETWORK:-regtest}
       LIGHTNINGD_CHAIN: grs
+      LIGHTNINGD_ALIAS: ${LIGHTNING_ALIAS}
+      LIGHTNINGD_ANNOUNCEADDR: ${BTCPAY_ANNOUNCEABLE_HOST}
+      LIGHTNINGD_PORT: 9735
       LIGHTNINGD_EXPLORERURL: "http://nbxplorer:32838/"
       LIGHTNINGD_OPT: |
         bitcoin-datadir=/etc/groestlcoin
         bitcoin-rpcconnect=groestlcoind
-        bitcoin-rpcport=43782
-        log-level=debug
-        announce-addr=${BTCPAY_HOST}:9738
-        bind-addr=0.0.0.0:9735
-        network=${NBITCOIN_NETWORK:-regtest}
-        alias=${LIGHTNING_ALIAS}
     volumes:
       - "clightning_groestlcoin_datadir:/root/.lightning"
       - "groestlcoin_datadir:/etc/groestlcoin"
       - "nbxplorer_datadir:/root/.nbxplorer"
     ports:
-      - "9738:9735"
+      - "9735:9735"
```

Comment:
this will conflict with mainnet btc

Reply:
I am aware. However several LN implementations already conflict with mainnet btc port 9735:
Litecoin c-lightning mainnet: https://github.com/btcpayserver/btcpayserver-docker/blob/master/docker-compose-generator/docker-fragments/litecoin-clightning.yml#L13
Bitcoin Gold LND mainnet: https://github.com/btcpayserver/btcpayserver-docker/blob/master/docker-compose-generator/docker-fragments/bgold-lnd.yml#L18
Groestlcoin eclair mainnet: https://github.com/btcpayserver/btcpayserver-docker/blob/master/docker-compose-generator/docker-fragments/groestlcoin-eclair.yml#L22

Maybe we should dedicate fixed LN ports for coins? (for example btc 9735, GRS 9736, LTC 9737 etc etc)
How do you want me to proceed?